src/@types/cidr-regex/index.d.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ *
+ */
+
+declare module "cidr-regex";
```

Comment:
in the functions project we use `cidr-matcher` and [we have TS types too](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/types/cidr-matcher.d.ts), you should use it here as well

Reply:
@lussoluca ok, could you pls add more specific type defs for the module if possible? (i.e. methods)

@gunzip AFAIK unfortunately right now there's no data shared between Azure and the Kubernetes cluster and we can't limit the source IP at the network level (we need the ingress to be able to receive traffic from any IP)